cmd/juju/caas/add.go

```diff
@@ -44,10 +46,17 @@ Adds a k8s endpoint and credential to Juju.`[1:]
 
 var usageAddCAASDetails = `
 Creates a user-defined cloud and populate the selected controller with the k8s
-cloud details.
+cloud details. Speficify non default kubeconfig file location using $KUBECONFIG
+environment variable or pipe in file content from stdin. The config file
+can contain definitions for different k8s clusters, use --context-name to pick
```

Comment:
I thought we were going to make this --cluster-name
Or even --cloud-name (since the metadata is added to the list of juju clouds)

Reply:
Actually, not --cloud-name as that's an arg. I think --cluster-name?

Reply:
Yeah, `cluster-name` is easy to understand. 
But the `cluster-name` and `context-name` in `~/.kube/config` file are different. And the existing code is using the context-name to find related records from `Contexts`. -> 
```
type ClientConfig struct {
	Type           string
	Contexts       map[string]Context
	CurrentContext string
	Clouds         map[string]CloudConfig
	Credentials    map[string]cloud.Credential
}
```
Do u think we should change it to use `cluster-name` by finding the related `context-name` according to the `cluster-name`?